scripts/scil_compute_seed_by_labels.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Computes the information from the seeding map for each cortical region
+(corresponding to an atlas) associated with a specific bundle.
+Here we want to estimate the seeding attribution to cortical area
+affected by the bundle
+"""
+
+import argparse
+import json
+
+import nibabel as nib
+import numpy as np
+
+from scilpy.io.image import get_data_as_label
+from scilpy.io.utils import (add_overwrite_arg, assert_inputs_exist)
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(
+        description=__doc__,
+        formatter_class=argparse.RawTextHelpFormatter)
+    p.add_argument('labels',
+                   help='Path of the input atlas file.')
+    p.add_argument('labels_lut',
+                   help='Path of the lookup table file, '
+                        'used to name the regions of interest.')
+    p.add_argument('seed_maps',
+                   help='Path of the input seeding file')
+
+    add_overwrite_arg(p)
+
+    return p
+
+
+def main():
+    parser = _build_arg_parser()
+    args = parser.parse_args()
+
+    assert_inputs_exist(parser, args.labels, args.seed_maps)
+
+    # Load atlas image
+    label_img = nib.load(args.labels)
+    label_img_data = get_data_as_label(label_img)
+
+    # Load atlas lut
+    with open(args.labels_lut) as f:
+        label_dict = json.load(f)
+    (label_indices, label_names) = zip(*label_dict.items())
+
+    # Load seed image
+    seed_img = nib.load(args.seed_maps)
+    seed_img_data = seed_img.get_fdata(dtype=np.float32)
+
+    for label, name in zip(label_indices, label_names):
+        if int(label) != 0:
```

Comment:
Instead of casting it multiple times, would be better to either cast it once at the beginning of the loop (before the `if`), or preconvert all indices.